air/src/range.rs

```diff
@@ -0,0 +1,68 @@
+use vm_core::RANGE_CHECK_TRACE_OFFSET;
+
+use super::{
+    utils::is_binary, Assertion, EvaluationFrame, Felt, FieldElement, TransitionConstraintDegree,
+};
+
+// CONSTANTS
+// ================================================================================================
+
+pub const COL_IDX_T: usize = RANGE_CHECK_TRACE_OFFSET;
+pub const COL_IDX_S0: usize = RANGE_CHECK_TRACE_OFFSET + 1;
+pub const COL_IDX_S1: usize = RANGE_CHECK_TRACE_OFFSET + 2;
+pub const COL_IDX_V: usize = RANGE_CHECK_TRACE_OFFSET + 3;
```

Comment:
nit: would prefer to put the name of the column first (I think it makes reading constraints a bit easier). For example: `T_COL_IDX` rather than `COL_IDX_T`.